datastore/integration-test/test_knmi.py

```diff
@@ -46,7 +46,10 @@ def test_find_series_single_station_all_parameters(grpc_stub):
     request = dstore.GetObsRequest(filter=dict(platform=dstore.Strings(values=["0-20000-0-06260"])))
     response = grpc_stub.GetObservations(request)
 
-    assert len(response.observations) == 42  # Station 06260 doesn't have all parameters
+    # HACK: data-loader which straight to datastore doesn't drop the duplicates
+    assert (
+        len(set([obs.ts_mdata.parameter_name for obs in response.observations])) == 34
+    )  # Station 06260 doesn't have all parameters
```

Comment:
Not clear to me why this is needed.

Reply:
`client_knmi_station.py` is inserting `cloud_base_altitude:2.0:point:PT0S` and `cloud_area_fraction:2.0:point:PT0S` twice to the system from the same station through `grpc`. This cannot happen with `ingest`, since `ingest` overrides duplicates. 

So even if the `api` doesn’t return those observations twice with the KNMI testdata, they are still there and are included in the `GetObservations` response. So if we don’t want to use `len` in the test, the other way would be to alter the KNMI `netcdf` files to not have duplicate observations on that station and if we go that route, I think we should take duplicates out of every station to be consistent.

Reply:
I think it is not so much that the there are duplicate observations in the netcdf, but that several observations map to the same "parameter_name". Isn't the best way to solve this to remove the parameters `nc1, nc2, nc3, hc1, hc2 and hc3` from `data-loader/parameters.py`?

Reply:
These went completely under my radar. Decided to drop `nc` and `hc` so I don't need to alter the test data and took the `set` out from this test: https://github.com/EURODEO/e-soh/pull/209/commits/086e592c27bdf770029569d1cf44e5fe40908a0a

Reply:
I am surprised this works. The `nc1`, `nc2` and `nc3` still insert data in the same spot, right?

Reply:
Yes, they go to the exact same spot.

The problem was because `nc1`, `nc2` and `nc3` have the `first`, `second` and `third layer` in their _long_name_ (_example: `nc1` = Cloud Amount First Layer_). The `layers` have no real meaning, but they turn to levels of `0.0`, `1.0` and `2.0` as a previous hack: 
```
# We want "level" to be numeric, so get rid of "first", "second" and "third".
# Note that this level has no meaning, it is just a hack for this test dataset
    if level == "first":
        level = "0"
    if level == "second":
        level = "1.0"
    if level == "third":
        level = "2.0"
 ```

The `nc` is _Total cloud cover_ by its _long_name_ and since it doesn't contain any level information, it used the "default" level of `2.0` with `generate_parameter_name` function, so it clashed with `nc3`. Same applies to `hc` and `hc3`

Reply:
Ah, thanks for clarifying.

Can we just drop nc1,2,3 and hc1,2,3 and get rid of all the level hacks for those? And keep nc and hc instead? Or is that a lot of refactoring in the output files? It would make it easier to understand when doing maintenance in the future.

Reply:
Those are only used in a few output files, so we can do that quite easily. 

I will do some other adjustments to the `parameters.py` also, instead of:
```
# Drop parameters that are not CF compliant
                if not standard_name or standard_name in ["precipitation_duration", "rainfall_duration"]:
                    continue
```

Reply:
Done in: https://github.com/EURODEO/e-soh/pull/209/commits/fef62dffc7612ad5370d1c0da1e432fec773c033